tests/functional/store_test_failures_tests/test_store_test_failures.py

```diff
@@ -0,0 +1,152 @@
+import pytest
+
+from dbt.tests.util import (
+    check_relations_equal,
+    run_dbt,
+)
+
+from tests.functional.store_test_failures_tests.fixtures import (
+    seeds__people,
+    seeds__expected_accepted_values,
+    seeds__expected_failing_test,
+    seeds__expected_not_null_problematic_model_id,
+    seeds__expected_unique_problematic_model_id,
+    properties__schema_yml,
+    models__problematic_model,
+    models__fine_model,
+    models__file_model_but_with_a_no_good_very_long_name,
+    tests__failing_test,
+    tests__passing_test,
+)
+
+# used to rename test audit schema to help test schema meet max char limit
+# the default is _dbt_test__audit but this runs over the postgres 63 schema name char limit
+# without which idempotency conditions will not hold (i.e. dbt can't drop the schema properly)
+TEST_AUDIT_SCHEMA_SUFFIX = "dbt_test__aud"
+
+
+class TestStoreTestFailures:
```

Comment:
nit: should this be a Base class instead?

Reply:
I assume this is fine because we aren't expecting adapters to implement this test. @VersusFacit can you confirm?

Reply:
Great call, Matt. Doing this before hitting merge.

Reply:
As for your comment, Colin, that's right, but it was an oversight when renaming the original base class. All it would do is be silently incorrect since pytest would look for a test method and find nothing.